install/syndesis-privileged.yml

```diff
@@ -1821,3 +1828,181 @@ objects:
         persistentVolumeClaim:
           claimName: syndesis-upgrade
       restartPolicy: Never
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: syndesis-public-oauthproxy
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+      syndesis.io/component: syndesis-public-oauthproxy
+- apiVersion: authorization.openshift.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: system:auth-delegator
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+      syndesis.io/component: syndesis-public-oauthproxy
+  subjects:
+  - kind: ServiceAccount
+    name: syndesis-public-oauthproxy
+    namespace: ${NAMESPACE}
+  roleRef:
+    kind: ClusterRole
+    name: system:auth-delegator
+    apiGroup: rbac.authorization.k8s.io
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+      syndesis.io/component: syndesis-public-oauthproxy
+    annotations:
+      service.alpha.openshift.io/serving-cert-secret-name: syndesis-public-oauthproxy-tls
+    name: syndesis-public-oauthproxy
+  spec:
+    ports:
+    - port: 8443
+      protocol: TCP
+      targetPort: 8443
+    selector:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/component: syndesis-public-oauthproxy
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+    annotations:
+      console.alpha.openshift.io/overview-app-route: "true"
+    name: syndesis-public-api
+  spec:
+    host: ${PUBLIC_API_ROUTE_HOSTNAME}
```

Comment:
Can't we just reuse the original ROUTE_NAME ?

Reply:
Had some issues making this work with OpenShift router. Will investigate in the future on how to setup a route to two oauth proxies and filter by uri path.